src/services/member/repository.ts

```diff
@@ -72,6 +75,10 @@ export const MemberRepository = AppDataSource.getRepository(Member).extend({
       newData.extra = Object.assign({}, body?.extra, body.extra);
     }
 
+    if (typeof body.enableSaveActions === 'boolean') {
```

Comment:
Can this really be undefined or null? I suppose you can only send a boolean value from the endpoint. Then I guess the service should provide a valid value.

Reply:
Yes but because it is a patch, the user can do a partial update of the member. That means enableSaveActions could be missing in the request body.